### PR TITLE
using things from @Shared/UserEnums

### PR DESCRIPTION
**Pull request by lisalluo**

### CODE REVIEW

File: src/client/pages/UserPage/components/UserProfile.tsx [hunk 1]
```diff
@@ -29,40 +30,26 @@ type Props = InjectedFormProps<UserProfileFormData, UserProfileProps> & UserProf
...
-    const genders = [
-      'Male', 'Female', 'Non-Binary', 'Transgender',
-      'I prefer not to say', 'Other',
-    ];
-
     return (
       <select {...input} className={className}>
-        {genders.map((gender, i) =>
-          <option key={i} value={gender}>{gender}</option>)}
+        {UserGenderOptions.map((UserGenderOptions, i) =>
```
Comment (subhankar-panda) [inline src/client/pages/UserPage/components/UserProfile.tsx]: You're resetting UserGenderOptions in the first arg of the map here - can we keep it `gender` as it was before?. (note: the previous code is *not* `genders.map((genders, i)...`) which i believe is the source of confusion

File: src/client/pages/UserPage/components/UserProfile.tsx [hunk 2]
```diff
@@ -29,40 +30,26 @@ type Props = InjectedFormProps<UserProfileFormData, UserProfileProps> & UserProf
...
-    const pronouns = [
-      'He/Him/His', 'She/Her/Hers', 'They/Them/Theirs',
-      'I prefer not to say', 'Other',
-    ];
-
     return (
       <select {...input} className={className}>
-        {pronouns.map((pronouns, i) =>
-          <option key={i} value={pronouns}>{pronouns}</option>)}
+        {UserPronounOptions.map((UserPronounOptions, i) =>
```
Comment (subhankar-panda) [inline src/client/pages/UserPage/components/UserProfile.tsx]: same

File: src/client/pages/UserPage/components/UserProfile.tsx (hunk 1 above)
Comment (lisalluo) [reply src/client/pages/UserPage/components/UserProfile.tsx]: good catch ty

File: src/client/pages/UserPage/components/UserProfile.tsx (hunk 2 above)
Comment (lisalluo) [reply src/client/pages/UserPage/components/UserProfile.tsx]: another good catch ty

**Pull request merged**